src/core/email-client.js

```diff
@@ -53,20 +68,62 @@ function getFreshImap (cb) {
 }
 
 class EmailClient {
-    constructor () {
+    //constructor takes the full name of user as String
+    constructor (name) {
         var self = this;
+        self.name = name; 
```

Comment:
Need function for updating the name in case it changes or wasn't available on initialization